tensorflow/tools/ci_build/Dockerfile.rocm

```diff
@@ -58,23 +58,27 @@ RUN apt-get update --allow-insecure-repositories && DEBIAN_FRONTEND=noninteracti
 RUN apt-get update --allow-insecure-repositories && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
     rocm-dev rocm-libs rocm-utils rocm-cmake \
+    rocfft miopen-hip miopengemm rocblas hipblas rocrand \
     rocm-profiler cxlactivitylogger && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
-# rocfft miopen-hip miopengemm rocblas hipblas rocrand \
 
 # Build HCC from source, cleanup the default HCC to avoid issue
 #RUN rm -rf /opt/rocm/hcc-1.0 && rm -rf /opt/rocm/lib/*.bc
-#RUN cd $HOME && git clone --recursive -b clang_tot_upgrade https://github.com/RadeonOpenCompute/hcc.git
+#RUN cd $HOME && git clone --recursive -b <> https://github.com/RadeonOpenCompute/hcc.git
 #RUN cd $HOME && mkdir -p build.hcc && cd build.hcc && cmake -DCMAKE_BUILD_TYPE=Release ../hcc
 #RUN cd $HOME/build.hcc && make -j $(nproc) && make package && dpkg -i hcc*.deb
 #RUN ln -s /opt/rocm/hcc-1.0 /opt/rocm/hcc
 
 # Build HIP from source
-RUN cd $HOME && git clone https://github.com/ROCm-Developer-Tools/HIP.git && \
-    mkdir HIP/build && cd HIP/build && \
-    git checkout -b 2019-03-29 4b7177ac4225c32ed0b02096a83b78fab0b9347a && \
-    cmake .. && make package -j $(nproc) && dpkg -i ./hip*.deb
+# roc-2.3.x branch as base + cherry-pick the merge commit for PR 981 (hipModuleGetGlobal fix)
+# the git config is required for the git cherry-pick command...
+RUN cd $HOME && git clone -b roc-2.3.x https://github.com/ROCm-Developer-Tools/HIP.git && \
+    cd $HOME/HIP && \
+    git config --global user.name "Deven Desai" && \
+    git config --global user.email "deven.desai.amd@gmail.com" && \
```

Comment:
@deven-amd how about `git cherry-pick --no-commit`

Reply:
that should work too...will make that change. let me know if you want it in this PR, or I can make it a part of the next PR

Reply:
let's make it another PR